src/CookiePlugin.php

```diff
@@ -109,11 +109,12 @@ private function createCookie(RequestInterface $request, $setCookie)
 
             switch (strtolower($key)) {
                 case 'expires':
-                    $expires = \DateTime::createFromFormat(DATE_COOKIE, $value);
+                    $expires = \DateTime::createFromFormat(DATE_COOKIE, $value)->getTimestamp()
```

Comment:
That would mean that the expires header sets, the actual expires header in the Cookie object (8th param). I do think this would be neat but its use is deprecated?